tests/test_opening.py

```diff
@@ -0,0 +1,62 @@
+from fixtures import *  # noqa: F401,F403
+from fixtures import TEST_NETWORK
+from utils import (
+    only_one, wait_for, sync_blockheight, EXPERIMENTAL_FEATURES
+)
+
+import unittest
+
+
+@unittest.skipIf(TEST_NETWORK != 'regtest', 'elementsd doesnt yet support PSBT features we need')
+@unittest.skipIf(not EXPERIMENTAL_FEATURES, "dual-funding is experimental only")
+def test_v2_rbf(node_factory, bitcoind, chainparams):
+    l1, l2 = node_factory.get_nodes(2,
+                                    opts=[{'dev-force-features': '+223'},
+                                          {'dev-force-features': '+223'}])
+
+    l1.rpc.connect(l2.info['id'], 'localhost', l2.port)
+    amount = 2**24
+    chan_amount = 100000
+    bitcoind.rpc.sendtoaddress(l1.rpc.newaddr()['bech32'], amount / 10**8 + 0.01)
+    bitcoind.generate_block(1)
+    # Wait for it to arrive.
+    wait_for(lambda: len(l1.rpc.listfunds()['outputs']) > 0)
+
+    res = l1.rpc.fundchannel(l2.info['id'], chan_amount)
+    chan_id = res['channel_id']
+    vins = bitcoind.rpc.decoderawtransaction(res['tx'])['vin']
+    assert(only_one(vins))
+    prev_utxos = ["{}:{}".format(vins[0]['txid'], vins[0]['vout'])]
+
+    # Check that we're waiting for lockin
+    l1.daemon.wait_for_log(' to DUALOPEND_AWAITING_LOCKIN')
+
+    next_feerate = l1.rpc.openchannel_next_feerate(chan_id)['next_feerate']
+
+    # Initiate an RBF
+    startweight = 42 + 172  # base weight, funding output
+    initpsbt = l1.rpc.utxopsbt(chan_amount, next_feerate, startweight,
+                               prev_utxos, reservedok=True,
+                               min_witness_weight=110,
+                               excess_as_change=True)
+
+    # Do the bump
+    bump = l1.rpc.openchannel_bump(chan_id, chan_amount, initpsbt['psbt'])
+
+    update = l1.rpc.openchannel_update(chan_id, bump['psbt'])
```

Comment:
It seems weird to me that openchannel_update doesn't take an explicit feerate?

Reply:
The `openchannel_update` refers to updating the *PSBT* that's in progress -- it's how you can add/remove inputs and outputs to the existing PSBT while the channel funding tx negotiation is ongoing.

Feerate's already set at this point, we'd have to add a new spec message to allow you to update it here...

As is, your options for changing a feerate would be to:
  - if first funding tx attempt, abort open, re-try openchannel
  - if first funding tx, finish open, init RBF (with automatic 25% fee bump)
  - if RBF, finish open, init RBF (with automatic 25% fee bump)
  - if RBF, abort / close channel by publishing the commitment tx, reinit channel with openchannel